test/e2e/knative_eventing_test.go

```diff
@@ -44,16 +36,29 @@ func TestKnativeEventing(t *testing.T) {
 	})
 
 	t.Run("verify correct deployment shape", func(t *testing.T) {
+		var eventingDeployments = []test.Deployment{
+			{Name: "eventing-controller"},
+			{Name: "eventing-webhook"},
+			{Name: "imc-controller"},
+			{Name: "imc-dispatcher"},
+			{Name: "mt-broker-controller"},
+			{Name: "mt-broker-filter"},
+			{Name: "mt-broker-ingress"},
+		}
+		if err := v1beta1.UpdateEventingExpectedScale(caCtx,
+			eventingNamespace, eventingNamespace, eventingDeployments, ptr.Int32(eventingHaReplicas)); err != nil {
+			t.Fatalf("Failed to update deployment scale: %v", err)
+		}
 		// Check the desired scale of deployments in the knative eventing namespace
-		for _, deployment := range []string{"eventing-controller", "eventing-webhook", "imc-controller", "imc-dispatcher", "mt-broker-controller"} {
-			if err := test.CheckDeploymentScale(caCtx, eventingNamespace, deployment, eventingHaReplicas); err != nil {
-				t.Fatalf("Failed to verify default HA settings for %q: %v", deployment, err)
+		for _, deployment := range eventingDeployments {
```

Comment:
This now also included `mt-broker-filter` and `mt-broker-ingress`. Are they always there?

Reply:
Previously it was not there because this part was checking that the deployment had 2 replicas. The broker-filter and broker-ingress used to have just one replica. So, we were only checking that those deployments exist but didn't verify the scale==2. Recently, these were made HA as well. They have two replicas now.
With our KnativeKafka config these two are there.